lib/sneakers/version.rb

```diff
@@ -1,3 +1,3 @@
 module Sneakers
-  VERSION = "2.12.0"
+  VERSION = "2.13.0"
```

Comment:
This version is not released and they already added new features to this version, wdyt if we use something like

```suggestion
  VERSION = "2.12.0patch1"
```

To not accidentally update to the base version and force to use the fork in services?

Reply:
Very good point!